### PR TITLE
Documentation update.

### PR DESCRIPTION
**Pull request by biscuitWizard**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -18,11 +18,15 @@ This will take a while to download, but it provides an easier method for updatin
...
+A database will also be required to enable persistence. You can get MySQL from [here](https://dev.mysql.com/downloads/mysql/) or MariaDB from [here](https://mariadb.org/download/). After installing MySQL, it will be necessary to connect to your local database using a client.
 
-    saving nebula.dmb (DEBUG mode)
+HeidiSQL is available [here](https://www.heidisql.com/download.php) or alternatively it's possible to use MySQL workbench available [here](https://dev.mysql.com/downloads/workbench/). Once that's downloaded, installed, and connected to the database. Create a database called **persistence** and execute to **schema.sql** which is available at the base level of this repository.
+
+This is a sourcecode-only release, so the next step is to compile the server files.  Open `baystation12.dme` by double-clicking it, open the Build menu, and click compile.  This'll take a little while, and if everything's done right you'll get a message like this:
```
Comment (Andrew-Fall) [inline docs/installation.md]: Not bay, nebula

File: docs/installation.md [hunk 2]
```diff
@@ -18,11 +18,15 @@ This will take a while to download, but it provides an easier method for updatin
...
+
+    saving baystation12.dmb (DEBUG mode)
```
Comment (Andrew-Fall) [inline docs/installation.md]: Not bay, nebula

File: docs/installation.md [hunk 3]
```diff
@@ -18,11 +18,15 @@ This will take a while to download, but it provides an easier method for updatin
...
     
-    nebula.dmb - 0 errors, 0 warnings
+    baystation12.dmb - 0 errors, 0 warnings
```
Comment (Andrew-Fall) [inline docs/installation.md]: Not bay, nebula

File: docs/installation.md [hunk 4]
```diff
@@ -34,13 +38,17 @@ Copy the contents of the `/config/examples` folder into `/config`. You will now
...
 
     byondkey - Rank
 
 where the BYOND key must be in lowercase and the admin rank must be properly capitalised.  There are a bunch more admin ranks, but these two should be enough for most servers, assuming you have trustworthy admins.
 
-To start the server, run Dream Daemon and enter the path to your compiled `nebula.dmb` file.  Make sure to set the port to the one you  specified in the `config.txt`, and set the Security box to 'Trusted' so you don't have to confirm access to every single configuration and storage file for the server.  Then press GO and the server should start up and be ready to join.
+To start the server, run Dream Daemon and enter the path to your compiled `baystation12.dmb` file.  Make sure to set the port to the one you  specified in the `config.txt`, and set the Security box to 'Trusted' so you don't have to confirm access to every single configuration and storage file for the server.  Then press GO and the server should start up and be ready to join.
```
Comment (Andrew-Fall) [inline docs/installation.md]: Not bay, nebula

File: docs/serialization.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+### FLATTEN MODE
+Flatten mode is a special flag that can be added to an entry in `saved_vars.json`. When `"flatten": true` the object will be serialized into json on the database, both increasing speed, efficiency, and stability of the saves. As well as preserving indexes safely. There are a few caveats to flatten mode that are important to keep in mind.
+
+* Flatten mode can only serialize text, numbers, nulls, and lists.
+* Flatten mode respects the `saved_vars.json` list and checks for all the same checks when deciding if a var should be serialized.
+* Do not flatten objects with references to other objects.
+* Do not flatten objects with paths, files, matrixes, and so on.
+
+Flatten is a *very* simple system and be overwhelmed easily. `/datum/gas_mixture` for example was a great candidate for flatten, but `/mob` obviously is not. Use responsibly.
```
Comment (Andrew-Fall) [inline docs/serialization.md:19]: Is this still valid after gas changes?

**Review by Andrew-Fall — CHANGES_REQUESTED** [review]

File: docs/serialization.md (hunk 5 above)
Comment (biscuitWizard) [reply docs/serialization.md:19]: yes, gas is still text because of wrappers now.

**Review by Andrew-Fall — APPROVED** [review]

**Pull request merged**